### PR TITLE
Feat: Unaffordable appeal cost on KlerosPOCCourt when there is no parent court

### PR DESCRIPTION
**Pull request by ferittuncer**
Related to #47 

Please review @clesaege  @epiqueras

### CODE REVIEW

**Comment by ferittuncer** [comment]
039b896 shouldn't be here, I'm fixing it.

**Comment by ferittuncer** [comment]
Fixed `appeal` function (forgotten replacement of `maxLocalAppeals`) and amended the commit @epiqueras

File: contracts/POC/KlerosPOCCourt.sol [hunk 1]
```diff
@@ -58,10 +58,12 @@ contract KlerosPOCCourt is ArbitratorCourt, KlerosPOC {
      *  @return _fee The appeal cost.
      */
     function appealCost(uint256 _disputeID, bytes _extraData) public constant returns(uint256 _fee) {
-        if (disputes[_disputeID].appeals < maxLocalAppeals) { // Will we stay under max local appeals?
+        if (disputes[_disputeID].appeals < maxAppeals) { // Will we stay under max local appeals?
             return super.appealCost(_disputeID, _extraData); // Regular appeal cost
+        } else {
+            if (parent._address == address(0) && bytes(parent.name).length == 0) return NON_PAYABLE_AMOUNT;
```
Comment (epiqueras) [inline contracts/POC/KlerosPOCCourt.sol]: No need to check `parent.name`.

File: contracts/POC/KlerosPOCCourt.sol [hunk 2]
```diff
@@ -58,10 +58,12 @@ contract KlerosPOCCourt is ArbitratorCourt, KlerosPOC {
...
+            return parent._address.arbitrationCost(_extraData); // `parent` arbitration cost
         }
```
Comment (epiqueras) [inline contracts/POC/KlerosPOCCourt.sol]: Remove blank line.

**Comment by epiqueras** [comment]
Looks good. Just some minor tweaks needed.

I also think we should refactor the files a bit:

`Kleros.sol` inherits from `Arbitrator.sol`.
`KlerosCourt.sol` inherits from `Kleros.sol`.
`KlerosPOC.sol` inherits from `Kleros.sol`.
`KlerosPOCCourt.sol` inherits from `KlerosCourt.sol && KlerosPOC.sol`.

**Comment by epiqueras** [comment]
@ferittuncer Yes. I updated my comment.

**Comment by ferittuncer** [comment]
I refactored as you suggested @epiqueras . Now we have an empty `KlerosCourt`. Do you think that we should move implementation of `appeal` and `appealCost` to `KlerosCourt` from  `KlerosPOCCourt` ?

**Comment by epiqueras** [comment]
@ferittuncer Nice, yes.

**Pull request merged**